starter-core/src/main/java/io/micronaut/starter/feature/database/r2dbc/R2dbc.java

```diff
@@ -46,9 +50,11 @@ public class R2dbc implements R2dbcFeature {
     private static final String PASSWORD_KEY = PREFIX + "password";
 
     private final DatabaseDriverFeature defaultDbFeature;
+    private final Hikari hikari;
 
-    public R2dbc(DatabaseDriverFeature defaultDbFeature) {
+    public R2dbc(DatabaseDriverFeature defaultDbFeature, Hikari hikari) {
```

Comment:
This is a breaking change. Can you annotate the new constructor with `@Inject` and keep the old one with `@Deprecated(forRemoval=True)`

Reply:
ping @timyates

Reply:
@sdelamo done!

Couldn't use `forRemoval` as thats a Java9 feature

Reply:
> Couldn't use forRemoval as thats a Java9 feature

👍🏻